tests/rest/client/test_read_marker.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2022 Beeper
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from twisted.test.proto_helpers import MemoryReactor
+
+import synapse.rest.admin
+from synapse.api.constants import EventTypes
+from synapse.rest import admin
+from synapse.rest.client import login, read_marker, register, room
+from synapse.server import HomeServer
+from synapse.util import Clock
+
+from tests import unittest
+
+one_hour_ms = 3600000
+one_day_ms = one_hour_ms * 24
```

Comment:
```suggestion
ONE_HOUR_MS = 3600000
ONE_DAY_MS = ONE_HOUR_MS * 24
```

(I think you copied this from `tests/rest/client/test_retation.py`, but no reason to propagate that.)

Reply:
https://github.com/matrix-org/synapse/pull/15464/commits/71e135349f950a663e25c36caa047f5decdd6c7a